### PR TITLE
Feat: Two Class YOLO

### PR DESCRIPTION
**Pull request by austin-nf**
## Problem Statement
The `yolov7-ros` repository needs to be updated to include the newly retrained weights to exclude classes not used by the vehicle during runtime (and suppressed during training).

## Change Log
- Add `yolov7-full-two-class-20260628.pt` weights file.

## Acceptance Criteria
- New weights file accessible.

### Dev Testing
See bench-marking [here](https://navflex.atlassian.net/wiki/spaces/NKB/pages/835911684/YOLO+v7+Model+History)

### CODE REVIEW

**Comment by nicholas-nf** [comment]
Functionality was also verified in two live dev runs and offline on the airbag regression test set. To clarify, this change excludes classes not used by the robot by suppressing them in training, although they are represented in the training set. If in the future we want to use yolo for carts or trailers, this model would not predict them.

**Comment by austin-nf** [comment]
Good clarification, edited in the summary.

**Review by rik-nf — APPROVED** [review]
